packages/wp-checkout/src/components/checkout-step.js

```diff
@@ -0,0 +1,90 @@
+/* @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import joinClasses from '../lib/join-classes';
+import { useLocalize } from '../lib/localize';
+
+export default function CheckoutStep( {
+	className,
+	stepNumber,
+	title,
+	onEdit,
+	collapsed,
+	children,
+} ) {
+	const classNames = collapsed
+		? [ className, 'checkout-step', 'checkout-step--is-collapsed' ]
```

Comment:
I think the host page could pass a space-delimited list for `className`. This is probably a good thing, maybe worth noting in the documentation.

Also- since we're using proptypes is it worth validating that `className` is a valid css class name?

Reply:
> I think the host page could pass a space-delimited list for className. This is probably a good thing, maybe worth noting in the documentation.

Good idea! Added to the README.

> since we're using proptypes is it worth validating that className is a valid css class name?

Hm... I suppose we could create a custom validator to do that. I'm not sure it's worth the effort, though. I think that if you're using `className` you probably know what kind of strings it supports. Also, CSS classNames can support a lot of kinds of strings.